src/Catel.MVVM/Catel.MVVM.Shared/MVVM/Converters/ValueConverterGroup.cs

```diff
@@ -35,13 +37,13 @@ namespace Catel.MVVM.Converters
     /// Original license: CPOL, compatible with the MIT license.
     /// </remarks>
     [ContentProperty("Converters")]
-    public class ValueConverterGroup : IValueConverter
+    public class ValueConverterGroup : System.Windows.Data.IValueConverter
```

Comment:
It's the first thing I tried, but it wouldn't compile due to the fact that the `Catel.MVVM.Converters` already contains an interface called `IValueConverter`. I don't think that interface can be deleted, because it would probably break existing code. Therefore I decided make the change as commited in the branch.